components/global/SiteHeader.vue

```diff
@@ -141,3 +127,17 @@
     },
   }
 </script>
+
+<style lang="postcss">
+  nav ul li a {
+    @apply p-6;
+  }
+
+  .nuxt-link-extact-active {
+    @apply border-accent opacity-100 text-primary-light font-semibold;
+  }
+
+  .height-limit {
+    max-height: 69px;
+  }
```

Comment:
Let's drop this altogether. It doesn't look to be doing much and removing it doesn't seem to break anything.

Reply:
I set this fixed height to accommodate to get rid of the 1px gap between the top hover boarder and the browser window.